knexfile.js

```diff
@@ -1,23 +1,13 @@
 require('dotenv').config();
 
-const log = require('./logger');
-
-const localPgConnection = {
-  host: 'localhost',
-  user: 'postgres',
-  password: 'key',
-  database: 'keylocal',
+const dbConnection = {
+  host: process.env.DATABASE_HOST,
+  port: process.env.DATABASE_PORT,
+  user: process.env.DATABASE_USER,
+  password: process.env.DATABASE_PASS,
+  database: process.env.DATABASE_NAME,
 };
 
-// Production database connection
-const dbConnection = process.env.DATABASE_URL || localPgConnection;
```

Comment:
I liked DATABASE_URL for local dev, it's only one env var to set instead of 5

Can we do:
```js
const dbConnection = process.env.DATABASE_URL || {
  host: process.env.DATABASE_HOST,
  port: process.env.DATABASE_PORT,
  user: process.env.DATABASE_USER,
  password: process.env.DATABASE_PASS,
  database: process.env.DATABASE_NAME,
};
```

Reply:
oops yeah that's how it was originally, good catch

Reply:
Reason for multiple env vars is so that the Docker Compose config can use them and spin up a local database with the same connection/credentials as the app.